src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,43 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, selectedTabId, setTab }) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map((tab) => {
+            const { id, title } = tab;
+
+            const changeTab = () => {
+              if (selectedTab !== id) {
+                setTab(tab);
```

Comment:
setTab should be called with 'id', not the entire 'tab' object, because in your 'App' component you are storing the 'selectedTabId', not the entire tab.